xml/System.Net/IPAddress.xml

```diff
@@ -285,10 +298,25 @@
         <Parameter Name="scopeid" Type="System.Int64" Index="1" FrameworkAlternate="netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <param name="address">To be added.</param>
-        <param name="scopeid">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <param name="address">The byte span value of the IP address.</param>
+        <param name="scopeid">The long value of the scope identifier.</param>
+        <summary>Initializes a new instance of the <see cref="T:System.Net.IPAddress" /> class with the address specified as a byte span and the specified scope identifier.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+ This constructor instantiates an IPv6 address. The `scopeid` identifies a network interface in the case of a link-local address. The scope is valid only for link-local and site-local addresses.  
+  
+ The byte span is assumed to be in network byte order with the most significant byte first in index position 0.  
+  
+ ]]></format>
+        </remarks>
+        <exception cref="T:System.ArgumentException">
+          <paramref name="address" /> contains a bad IP address.</exception>
+        <exception cref="T:System.ArgumentOutOfRangeException">
+          <paramref name="scopeid" /> &lt; 0 
+- or - 
+<paramref name="scopeid" /> &gt; 0x00000000FFFFFFFF</exception>
```

Comment:
```suggestion
<paramref name="scopeid" /> &gt; 0x00000000FFFFFFFF.</exception>
```

Reply:
It's better to use code in the IntelliSense portions 

```suggestion
<paramref name="scopeid" /> &gt; <code>0x00000000FFFFFFFF</code></exception>
```

Reply:
> but do you need to do this?

Well, this means that even though the field is `long`, it cannot exceed the maximum value possible stored in 32 bits, that is `4294967295`, which is also `uint.MaxValue`.

I don't know what would be the best way to express that but I took the format from [one of its overloads](https://docs.microsoft.com/en-us/dotnet/api/system.net.ipaddress.-ctor?view=netcore-3.0#System_Net_IPAddress__ctor_System_Byte___System_Int64_).

Reply:
Sorry, I should have been more clear @Jozkee. I was asking if we need to format the number as @carlossanlop was suggesting.

Reply:
My bad, I forgot this was not a remark. Thanks for the correction, @mairaw.

Reply:
No, not needed. Feel free to ignore this suggestion, @Jozkee .